app/ydoc-server-polyglot/src/main.ts

```diff
@@ -1,8 +1,8 @@
 import { configureAllDebugLogs, docName, setupGatewayClient } from 'ydoc-server'
 
-const host = YDOC_HOST ?? 'localhost'
-const port = YDOC_PORT ?? 1234
-const debug = YDOC_LS_DEBUG ?? false
+const host = typeof YDOC_HOST == 'string' ? YDOC_HOST : 'localhost'
```

Comment:
Originally I was getting an error here:
```js
$ js
> YDOC_HOST ?? 'localhost'
ReferenceError: YDOC_HOST is not defined
        at <js> :program(<shell>:1:1:0-8)
```
with the new code I am getting `localhost`:
```js
$ js
> typeof YDOC_HOST == 'string' ? YDOC_HOST : 'localhost'
localhost
> 
```